index.html

```diff
@@ -65,20 +65,14 @@ <h2 class="subtitle">Who's in charge?</h2>
         </div>
 
         <div class="column">
-          <a href="https://gellat.io" target="_blank" rel="noopener noreferrer"><img src="/img/people/gellatio.jpg" class="someone admin"></a>
-          <h3>Gellatio</h3>
-          <span class="tag is-info">Admin</span>
-        </div> 
-
-        <div class="column">
           <img src="/img/people/ren.png" class="someone admin">
           <h3>Renekton</h3>
           <span class="tag is-info">Admin</span>
         </div>
 
         <div class="column">
           <a href="https://jaffasite.ga" target="_blank" rel="noopener noreferrer"><img src="/img/people/jaffa.png" class="someone admin"></a>
-          <h3><span title="AKA TheJaffaMeme">TheJaffaMan</span></h3>
+          <h3><span title="TheJaffaMeme">TheJaffaMeme</span></h3>
```

Comment:
Please remove the `title=" ... "` bit if its contents are the exact same as the text shown